README.md

```diff
@@ -49,29 +49,72 @@ polygon detection is performed and checked if the the number of sides is more th
 * This is not robust enough, the thresholds need to be adjusted always.
 * Doesnt work properly on real world data as there is lot of noise. 
 
-### Real World (Test Lot) --- YOLOv3-tiny (You Only Look Once)
-We used this approach for real world.
-TODO:write about it
-
-### Real World (Test Lot) --- SSD (Single Shot Detection)
+## SSD Approach (Single Shot Detection)
 We need to solve both object detection - where in the image is the object, 
 and object classification --- given detections on an image, classify traffic lights. 
 While there are teams who approached it as 2 separate problems to be solved, 
 recent advancements in Deep Learning has developed models that attempt to solve both at once.
 For example, SSD (Single Shot Multibox Detection) and YOLO (You Only Look Once).
+### Simulator (Highway) --- SSD Approach
+Here we experimented with Tensorflow Object Detection API, using pretrained models on COCO dataset, such as: "ssd_inception_v2_coco" and "ssd_mobilenet_v2_coco":
+* Testing the coco pretrained models without retraining on simulator images didn't lead to any success, since the simulated traffic lights look very different from real traffic lights, and hence we concluded, that if we were going to use this approach on the simulator, we would need a different model specifically retrained on the simulator images. 
+* So we decided to utilize transfer learning, and retrain the models on images extracted from the simulator, using 3 classes/labels only; Red, Yellow and Green.
+* We choose the "ssd_inception_v2_coco model", since it proved to be a good compromise between speed and accuracy, and retrained it on the simulator images dataset provided by Alex Lechner [here](https://github.com/alex-lechner/Traffic-Light-Classification/blob/master/README.md#1-the-lazy-approach).
+
+Sample dataset for simulator images
+![simulator training images](report/sim_visualization.png)
```

Comment:
Is it possible to annotate bounding boxes with label and confidence? It could be done in the future pull request, while #14 is ready.

Reply:
noted

Reply:
Isn't this a sample for the dataset images?....the annotation should be done in line: 75 & 103

Reply:
@Elgeweily, you are right. Let's place here just a few images from the simulator dataset.
```
red         yellow
green   no light
```